apps/src/codebridge/FileBrowser/styles/filebrowser.module.scss

```diff
@@ -74,13 +74,15 @@
 // (this allows it to take precedence over the component library button styles)
 button.button-kebab {
   display: none;
-  height: 20px;
```

Comment:
These were just in the wrong place. If we have this display:none, then definitionally none of these styles will apply🫠